From 268e104da51a7bc4dc6d127beab552a25858e268 Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 22 Jul 2016 15:22:16 +0000
Subject: [PATCH 0773/1076] MANTIS 40176 extension: Components for
 enabling/disabling TouchScreen and Display

- added enum displayState and CMD_SWITCH_DISPLAY to appcontrol.idl

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1053 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/appcontrol.idl | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl b/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
index b55456e8..c44fdfc2 100644
--- a/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
@@ -38,6 +38,12 @@ module appControl
       RES_1600x1200,
       RES_DEFAULT
     };
+    
+    enum displayState
+    {
+      DISPLAY_ON,
+      DISPLAY_OFF
+    };
 
     // deprecated for CMD_START used applicationStart, for CMD_SWITCH, CMD_KILL, CMD_SHOW use applicationId */
     struct application
@@ -75,7 +81,8 @@ module appControl
       CMD_KILLALL,        /**< terminates all applications and closes the associated desktops */
       CMD_SWITCHMAIN,     /**< Switches to the main desktop */
       CMD_SHOW,           /**< Switches to a specific desktop, but doesn't activate it (debug mode)*/
-      CMD_SET_RESOLUTION  /**< Set a screen resolution */
+      CMD_SET_RESOLUTION, /**< Set a screen resolution */
+      CMD_SWITCH_DISPLAY  /**< Switch display ON or OFF */
     };
 
     /** All device components inherit from the generic DeviceComponents interface */
-- 
2.41.0.windows.1

